src/state-summary/state-card-weather.html

```diff
@@ -0,0 +1,110 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../../bower_components/google-apis/google-legacy-loader.html">
+
+<dom-module id="state-card-weather">
+  <template>
+      <google-legacy-loader on-api-load="googleApiLoaded"></google-legacy-loader>
+    </template>
+  </template>
+</dom-module>
+
+<script>
+Polymer({
+  is: 'state-card-weather',
+
+  properties: {
+    stateObj: {
+      type: Object,
+      observer: 'dataChanged',
+    },
+
+    chartEngine: {
+      type: Object,
+    },
+
+    isAttached: {
+      type: Boolean,
+      value: false,
+      observer: 'dataChanged',
+    }
+  },
+
+  attached: function () {
+    this.isAttached = true;
+  },
+
+  dataChanged: function() {
+    this.drawChart();
+  },
+
+  getDataArray: function() {
```

Comment:
This function only cares about forecast instead of also including the current temperature.